### DISCUSSION TITLE
Powerline font rendering error

### DISCUSSION THREAD

**Discussion by OliverFM** [ohmyzsh/ohmyzsh#9825] [Help] [answered]
I have installed the Meslo fonts & can see all the symbols in the config, however when I run `cd` I get symbols that are incorrectly displayed. This is running on ubuntu 18.04 with the default terminal.

![Screenshot from 2021-04-09 11-41-18](https://user-images.githubusercontent.com/16237233/114168797-a58aa300-9928-11eb-8a00-e62cb862554b.png)

I have set powerline fonts to be enabled and with agnoster, for example, I can see the expected symbols rendered  yet there is still an issue with surrounding `echo`:
![Screenshot from 2021-04-13 11-50-33](https://user-images.githubusercontent.com/16237233/114541432-a2582580-9c4e-11eb-85cf-1f97fc26ddf0.png)

**Comment by mcornella** [comment, marked as answer]
Looks like you have a `preexec` hook function that's outputting a broken formatting string, that's where those `001B` boxes come from. Can you post the output of `which $preexec_functions` and `echo $TERM`?

**Reply by OliverFM** [reply]
Thanks -- I had opened an issue elsewhere and it turned out to be exactly that. I was setting `TERM` which I now no longer do. THanks for the help! :pray:
